src/test/java/com/baeldung/selenium/alertsandpopups/HandlePopupsUnitTests.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.selenium.alertsandpopups;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HandlePopupsUnitTests {
```

Comment:
These tests rely on a live website and so they can't be labelled as unit tests
```suggestion
public class HandlePopupsLiveTest {
```